packages/core/src/database/migrations/z1690911789-create-subgraphMicrocreditBorrowers.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+        return queryInterface.createTable('subgraph_microcredit_borrowers', {
+            id: {
+                type: Sequelize.INTEGER,
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true
+            },
+            userId: {
+                type: Sequelize.INTEGER,
+                allowNull: false,
+                references: {
+                    model: 'app_user',
+                    key: 'id',
+                },
+                onDelete: 'CASCADE'
+            },
```

Comment:
Instead, I suggest having an `address` field, that is actually the primary key and references to `app_user.address`. Any id as the primary key on this table is useless. Having the `userId` requires fetching user Ids from the address when updating, and I think it won't improve any aspect of the queries being done later on. But I might be wrong. What do you think?

Reply:
I thought a lot about using the `userId` or `address`. In the process to populate this table is much easier to use the `address`, but to list borrowers is better to use `userId` because the `microCreditBorrowers` table (and others) uses the user id as the reference of the user.
So following the pattern that we used for the other tables/associations I preferred the `userId` to facilitate on list borrowers.